cmeutils/structure.py

```diff
@@ -145,6 +147,8 @@ def gsd_rdf(
             rdf.compute(aq, neighbors=nlist, reset=False)
 
         normalization = post_filter / pre_filter if exclude_bonded else 1
+        normalization *= ab_ratio
```

Comment:
LGTM! I'm not sure theory-wise why this fixes it. @erjank Do you have a good explanation? If we're still stumped, we should reach out to Bradley or Vyas to help us understand.

Reply:
I think it's something related to the AABB query module, but I'm not that familiar with it. Maybe something related to `exclude_ii`? 

I left a comment on the issue:

'm not sure if this is related to the `post_filter` and `pre_filter` implementation. Those variables are only defined when `exclude_bonded=True`, and this issue occurs regardless of what is set for `exclude_bonded.`

I wonder if it is instead due to the the neighbor list query step. `system` is always created using `A_pos` and the query is always done for `B_pos` which probably explains why the behavior depends on what order `A_name` and `B_name` are defined.